.ci/scripts/conformance-scripts/run_tests.py

```diff
@@ -7,7 +7,7 @@
 
 algs_filename = "algorithms.txt"
 report_filename = "report.html"
-sklearn_version = "0.23.1" if len(sys.argv) == 1 else sys.argv[1]
+sklearn_version = "0.23.2" if len(sys.argv) == 1 else sys.argv[1]
```

Comment:
Can we download the latest release version automatically?

Reply:
We can download latest 0.23.X version, but if this version will change on 0.24, then we must manually fix it